WikilogUtils.php

```diff
@@ -210,7 +210,7 @@ public static function parsedArticle( Title $title, $feed = false ) {
 		$parser->startExternalParse( $title, $parserOpt, Parser::OT_HTML );
 
 		# Parse article.
-		$arttext = $article->getText();
+		$arttext = $article->getContent()->getNativeData();
```

Comment:
Please check, if the returned Content object is a TextContent (instanceof TextContent), as this is currently the implementation, which will return text for getNativeData(). Any other implementation will vary depending on the content model of the page.

Here in this case, I think it's best to just check if the content model is TextContent, and if not return default values (null or empty things).

Reply:
okay!